packages-ts/starknet-gauntlet-starkgate/src/commands/L1-bridge/withdraw.ts

```diff
@@ -0,0 +1,65 @@
+import {
```

Comment:
Please also add capability to call the [`initiate_withdraw`](https://github.com/starkware-libs/starkgate-contracts/blob/c08863a1f08226c09f1d0748124192e848d73db9/src/starkware/starknet/apps/starkgate/cairo/token_bridge.cairo#L179) on the L2 bridge contract.